src/consensus.rs

```diff
@@ -1265,6 +1534,24 @@ impl Consensus {
                 Ok(())
             }
             ConsensusCommand::StartNewSlot => self.start_new_slot(),
+            ConsensusCommand::TimeoutTick => match &self.bft_round_state.timeout_state {
+                ConsensusTimeoutState::Scheduled { timestamp }
+                    if get_current_timestamp() >= *timestamp =>
+                {
+                    // Trigger state transition to mutiny
+                    info!("Trigger timeout");
```

Comment:
would add more detail to log like slot & view
And maybe an emoji like ⏰ to see it easily in logs ? :)